CefSharp.Wpf/Experimental/ChromiumWebBrowserWithTouchSupport.cs

```diff
@@ -20,20 +20,84 @@ namespace CefSharp.Wpf.Experimental
     /// </summary>
     public class ChromiumWebBrowserWithTouchSupport : ChromiumWebBrowser
     {
+        #region Mouse
+        //Mouse, touch and sylus pen will Run OnMouseXXX.
+        //Use e.StylusDevice == null to ensure only mouse.
+
+        /// <summary>
+        /// Handles the <see cref="E:MouseButton" /> event.
+        /// </summary>
+        /// <param name="e">The <see cref="MouseButtonEventArgs"/> instance containing the event data.</param>
         protected internal override void OnMouseButton(MouseButtonEventArgs e)
         {
             // For mouse events from an actual mouse, e.StylusDevice will be null.
-            //For mouse events from touch and stylus, e.StylusDevice will not be null.
+            // For mouse events from touch and stylus, e.StylusDevice will not be null.
             // If we don't check if e.StylusDevice == null, touch scrolls will also select text.
             if (e.StylusDevice == null)
             {
                 base.OnMouseButton(e);
             }
         }
+
+        /// <summary>
+        /// Invoked when an unhandled <see cref="E:System.Windows.Input.Mouse.MouseMove" /> attached event reaches an element in its route that is derived from this class. Implement this method to add class handling for this event.
+        /// </summary>
+        /// <param name="e">The <see cref="T:System.Windows.Input.MouseEventArgs" /> that contains the event data.</param>
+        protected override void OnMouseMove(MouseEventArgs e)
+        {
+            if (e.StylusDevice == null)
```

Comment:
Please add comments

Reply:
I think this could be more descriptive,  explain in what case mouse and stylus events are present in the one event please.

Reply:
Using comment from OnMouseButton above I'd fine